dependencies.yaml

```diff
@@ -920,28 +921,32 @@ dependencies:
     common:
       - output_types: [conda, requirements, pyproject]
         packages:
-          - fastavro>=0.22.9
-          # TODO: Remove hypothesis pinning once https://github.com/HypothesisWorks/hypothesis/issues/4365 is resolved
-          - hypothesis<6.131.1
```

Comment:
Just noting that this is also being changed in https://github.com/rapidsai/cudf/pull/18537/files#diff-5475a6d76de4c506ee92cf6f941bba30a6a07b5881cfea531d42a6ec035095a6L934